stencil-library/src/components/business-details/shared-business-details.scss

```diff
@@ -0,0 +1,60 @@
+@import 'bootstrap-utilities';
+@import 'grid';
+
+:host {
+  @include host-base-styles;
+}
+
+// Detail Sections
+detail-section {
+  margin-bottom: 20px;
+
+  &::part(detail-section-title) {
+    font-size: 1.5rem;
+    margin-bottom: 15px;
+  }
+}
+
+// Detail Items
+detail-item {
+  margin-bottom: 10px;
+  gap: 10px;
+
+  &::part(detail-section-item-title),
+  &::part(detail-section-item-data) {
+    display: inline-block;
+  }
+
+  &::part(detail-section-item-title) {
+    font-weight: bold;
+    width: 150px;
+  }
+
+  &::part(detail-section-item-data) {
+    width: calc(100% - 150px);
+  }
+
+  .flex-1 {
+    flex: 1;
+  }
+
+  ::part(flex-1) {
+    flex: 1;
+  }
+}
+
+.flex-1 {
+  flex: 1;
+}
+
+::part(flex-1) {
+  flex: 1;
+}
+
+// Loading and Error States
+.loading-state,
+.error-state {
+  font-size: 1.2rem;
```

Comment:
Since these are shared components, the loading and error states, should these styles be directly applied to those so that everything is consistent? Also there are bootstrap utility classes for all three of these properties that we can use:
https://getbootstrap.com/docs/5.3/utilities/text/
https://getbootstrap.com/docs/5.3/utilities/spacing/